warehouse/query-core/src/main/java/datawave/query/jexl/JexlASTHelper.java

```diff
@@ -215,7 +216,9 @@ public static ASTJexlScript parseJexlQuery(String query) throws ParseException {
         } else {
             // Parse the original query
             try {
-                return parser.parse(null, jexlFeatures(), caseFixQuery, null);
+                // Using this constructor for the JexlInfo cuts parse time by 50%
+                JexlInfo jexlInfo = new JexlInfo("parseJexlQuery", 1,1);
+                return parser.parse(jexlInfo, jexlFeatures(), caseFixQuery, null);
```

Comment:
consider a utility method that binds in the JexlInfo correctly to reduce duplication?